Trello_items.py

```diff
@@ -0,0 +1,78 @@
+from secrets import KEY, TOKEN
+import requests
+
+todo_boardid = "5f6076c34c5f48265943e31e"
+todolistid = "5f6076e68cc021208da06d2b"
+donelistid = "5f6076e96994a166c05385a6"
+
+apiurl = "https://api.trello.com/1/"
+boardsurl = apiurl + 'boards/'
+getcardsonboardsurl = boardsurl + todo_boardid + '/cards'
+cardsurl = apiurl + 'cards/'
+
+def getcardsonlist_URL(cardid):
+    URL = cardsurl + cardid + "/list"
+    return URL
+def putcardsonlist_URL(cardid):
+    URL = cardsurl + cardid
+    return URL
+
+def buildquery(querytype, title=''):
+    if querytype == 'getitems':
+        return {'key' : KEY, 'token' : TOKEN}
+    elif querytype == "saveitem":
+        return {'key' : KEY, 'token' : TOKEN, "idList": donelistid}
+    elif querytype == "additem":
+        return {'key' : KEY, 'token' : TOKEN, "idList": "5f6076e68cc021208da06d2b", "name" : title}
```

Comment:
It's a bit messy to have this function which needs to know about all of the different queries.

It should just be something like
```
def build_auth_query():
    return {'key' : KEY, 'token' : TOKEN}
```
And then the other functions can add to it, e.g. inside save_item_trello:
```
    query = build_auth_query()
    query["idList"] = donelistid
    requests.put(...., params = query)
```